examples/ex14.cpp

```diff
@@ -44,6 +44,7 @@ int main(int argc, char *argv[])
    int order = 1;
    double sigma = -1.0;
    double kappa = -1.0;
+   double eta = 1.0;
```

Comment:
Are these the default values we want?

Reply:
I changed eta to be zero by default, so the previous behavior remains the same. This is probably preferable, since the BR2 terms can make the system harder to solve.